### PR TITLE
adding ability to get scores for simulated doublets

### PR DESCRIPTION
**Pull request by njbernstein**

### CODE REVIEW

File: scvi/external/solo/_model.py [hunk 1]
```diff
@@ -324,15 +324,19 @@ def train(
         return runner()
 
     @torch.no_grad()
-    def predict(self, soft: bool = True) -> pd.DataFrame:
+    def predict(
+        self, soft: bool = True, only_experimental_data: bool = True
```
Comment (adamgayoso) [inline scvi/external/solo/_model.py:328]:
```suggestion
        self, soft: bool = True, include_simulated_doublets: bool = False
```

File: scvi/external/solo/_model.py [hunk 2]
```diff
@@ -357,7 +361,11 @@ def auto_forward(module, x):
 
         label = self.adata.obs["_solo_doub_sim"].values.ravel()
         singlet_mask = label == "singlet"
```
Comment (adamgayoso) [inline scvi/external/solo/_model.py:363]:
```suggestion
        mask = label == "singlet" if not include_simulated_doublets else slice(None)
```

File: scvi/external/solo/_model.py [hunk 3]
```diff
@@ -357,7 +361,11 @@ def auto_forward(module, x):
...
-        preds = y_pred[singlet_mask]
+        if only_experimental_data:
+            preds = y_pred[singlet_mask]
+        else:
+            preds = y_pred
```
Comment (adamgayoso) [inline scvi/external/solo/_model.py:367]:
```suggestion
        preds = y_pred[mask]
```

Comment (adamgayoso) [reply scvi/external/solo/_model.py:367]: also need to use mask and not singlet mask in line 373 in pandas df creation

File: scvi/external/solo/_model.py [hunk 4]
```diff
@@ -324,15 +324,19 @@ def train(
...
+    ) -> pd.DataFrame:
         """
         Return doublet predictions.
 
         Parameters
         ----------
         soft
             Return probabilities instead of class label
-
+        only_experimental_data
```
Comment (adamgayoso) [inline scvi/external/solo/_model.py:337]: rename as above

**Review by adamgayoso — CHANGES_REQUESTED** [review]
Just a few chages, looks great though!

File: scvi/external/solo/_model.py [hunk 5]
```diff
@@ -324,15 +324,18 @@ def train(
         return runner()
 
     @torch.no_grad()
-    def predict(self, soft: bool = True) -> pd.DataFrame:
+    def predict(
+        self, soft: bool = True, include_simulated_doublets: bool = True
```
Comment (adamgayoso) [inline scvi/external/solo/_model.py:328]:
```suggestion
        self, soft: bool = True, include_simulated_doublets: bool = False
```
Default should be false right? I don't think most users care to see the sim doubs?

Comment (njbernstein) [reply scvi/external/solo/_model.py]: yup!

**Review by adamgayoso — APPROVED** [review]
LGTM!
